code/src/Core/PostActions/Catalog/NewItemGeneration/CreateSyncSummaryPostAction.cs

```diff
@@ -141,19 +134,14 @@ public override void Execute()
             GenContext.Current.MergeFilesFromProject.Clear();
         }
 
-        private static string GetLinkToProjectFile(string fileName, string filePath)
-        {
-            return $"* [{fileName}]({FormatFilePath(filePath)})";
-        }
-
         private static string GetCompareLink(string fileName, string filePath)
         {
-            return $"* {StringRes.SyncSummaryTempGenerationFile}: [{fileName}]({fileName}), {StringRes.SyncSummaryProjectFile}: [{fileName}]({FormatFilePath(filePath)})";
+            return $"* {StringRes.SyncSummaryTempGenerationFile}: [{fileName}]({fileName}), {StringRes.SyncSummaryProjectFile}: {GetMarkDownLinkToLocalFile(filePath)}";
         }
 
-        private static string FormatFilePath(string filePath)
+        private static string GetMarkDownLinkToLocalFile(string fileName)
         {
-            return $"about:/{filePath.Replace(" ", "%20").Replace(@"\", "/")}";
+            return $"* [{fileName}]({fileName})";
```

Comment:
This will point to the file in the temp generation folder although it is already included in the project. 
Are we sure that we want to open the temp file and not the already integrated file? I think that could be weird and lead to confusion and errors (what if the developer does not notice and edits the temp file instead of the file integrated in the project)